src/types/onyx/PolicyReportField.ts

```diff
@@ -0,0 +1,22 @@
+type PolicyReportFieldType = "text" | "date" | "dropdown";
+
+type PolicyReportField = {
+    /** Name of the field */
+    name: string;
+
+    /** Default value assigned to the field */
+    defaultValue: string;
+
+    /** Unique id of the field */
+    fieldId: string;
+
+    /** Position at which the field should show up relative to the other fields */
+    orderWeight: number;
+
+    /** Type of report field */
+    type: PolicyReportFieldType;
+};
```

Comment:
We will need a couple more here

`value`, and `values` (for dropdown values)

and then I'm still looking into this but possible `required` (looking into this today)

Reply:
We don't need the `required` key - I guess if the text_title exists it will be required

Reply:
@thienlnam Can we rename these in our code? I was thinking that it makes more sense to call it `required` and `options` instead of `deletable` and `values`?

Reply:
I mean we can update them just for the front-end but to modify these keys in the BE it will be more work than it is worth due to updating oldApp as well.

I will update these keys when we re-format them from onyx to be `required` and `options`

Reply:
Actually thinking about this more, I think this will just end up adding more confusion in the future with the disconnect between the keys

Deletable is also representative of the fields that no longer exist on old reports but no longer on the policy, and I don't know if we should just rename a single key (values is still pretty intuitive)

Reply:
Fine by me. I'll change this so that it matches what we have already.